multiapi-engine/src/test/resources/asyncapigenerator/testCustomValidators/customvalidator/PatternValidator.java

```diff
@@ -0,0 +1,25 @@
+package com.sngular.scsplugin.customvalidator.model.event.customvalidator;
+
+import java.util.Objects;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class PatternValidator implements ConstraintValidator<Pattern, String> {
+
+  private String regex;
+
+  @Override
+  public void initialize(Pattern constraintAnnotation) {
+    ConstraintValidator.super.initialize(constraintAnnotation);
+    this.regex = constraintAnnotation.regex();
+  }
+
+  @Override
+  public boolean isValid(String value, ConstraintValidatorContext constraintValidatorContext) {
+    if (Objects.isNull(value))
+      return true;
+
+    return value.matches(regex);
```

Comment:
With the "or" operator, if the first component is already true, the second (which could launch a NullPointerException) is not evaluated (and the exception won't be launched):
```return Objects.isNull(value) || value.matches(regex);```

Reply:
Done